<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetadataPrefsDialog</class>
 <widget class="QDialog" name="MetadataPrefsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tag Writer Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_13">
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBoxMP3">
       <property name="title">
        <string>MP3</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_11">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="mp3WriteID3v2">
            <property name="text">
             <string>Write ID3v2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mp3WriteID3v1">
            <property name="text">
             <string>Write ID3v1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mp3WriteAPEv2">
            <property name="text">
             <string>Write APEv2</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>ID3v2 Version</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxID3v2Version">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="mp3StripID3v2">
            <property name="text">
             <string>Strip ID3v2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mp3StripID3v1">
            <property name="text">
             <string>Strip ID3v1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mp3StripAPEv2">
            <property name="text">
             <string>Strip APEv2</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>ID3v1 character encoding(default is iso8859-1)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditID3v1Encoding"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBoxAPE">
         <property name="title">
          <string>APE</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="0" column="0">
           <widget class="QCheckBox" name="apeWriteID3v2">
            <property name="text">
             <string>Write ID3v2.4</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="apeWriteAPEv2">
            <property name="text">
             <string>Write APEv2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="apeStripID3v2">
            <property name="text">
             <string>Strip ID3v2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="apeStripAPEv2">
            <property name="text">
             <string>Strip APEv2</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxWavPack">
         <property name="title">
          <string>WavPack</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_1">
          <item row="0" column="1">
           <widget class="QCheckBox" name="wvWriteID3v1">
            <property name="text">
             <string>Write ID3v1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="wvWriteAPEv2">
            <property name="text">
             <string>Write APEv2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="wvStripAPEv2">
            <property name="text">
             <string>Strip APEv2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="wvStripID3v1">
            <property name="text">
             <string>Strip ID3v1</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MetadataPrefsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MetadataPrefsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
